release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2671,6 +2705,10 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 * Installation fails when installing {product-title} with the `bootMode` set to `UEFISecureBoot` on a node where Secure Boot is disabled. Subsequent attempts to install {product-title} with Secure Boot enabled will proceed normally. (link:https://issues.redhat.com/browse/OCPBUGS-19884)[*OCPBUGS-19884*])
 
 
+* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix.
```

Comment:
Can we update this with "or use Kuryr"? I think we opt out from Kuryr support with 17.1 given that it's going EOL in 4.15 and it's a significant testing cost. Nevertheless it does not work correctly at the moment due to Neutron issues and we can clearly state that.

Reply:
@maxwelldb I think something like following can be more specific about the Kuryr scenario:

For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents successful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. Also, there is not support for Kuryr installations on  {rh-openstack} 17.1.

Reply:
Works for me. 

```suggestion
* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix.r deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. Additionally, there is no support for Kuryr installations on {rh-openstack} 17.1.
```